src/main/scala/nl/biopet/utils/tool/ToolCommand.scala

```diff
@@ -37,11 +50,227 @@ trait ToolCommand[Args] extends Logging with ToolDocumentation {
+    * @return A HTML table as a string. The table contains newlines and indentation for readability.
+    */
+  def htmlTable(headers: List[String], body: List[List[String]]): String = {
+
+    // Validate that all rows have a length equal to the header
+    body.foreach(
+      row =>
+        require(
+          row.length == headers.length,
+          "Number of items in each row should be equal number of items in header."))
+
+    val table = new StringBuffer()
+
+    table.append("<table>\n")
+    table.append(
+      headers.mkString("\t<thead>\n\t\t<tr>\n\t\t\t<th>",
+                       "</th>\n\t\t\t<th>",
+                       "</th>\n\t\t</tr>\n\t</thead>\n"))
+    table.append("\t<tbody>\n")
+    for (row <- body) {
+      table.append(
+        row.mkString("\t\t<tr>\n\t\t\t<td>",
+                     "</td>\n\t\t\t<td>",
+                     "</td>\n\t\t</tr>\n"))
+    }
+    table.append("\t</tbody>\n")
+    table.append("</table>\n")
+    table.toString.replace("\t", "  ")
+  }
+
+  /**
+    * Generates a Markdown file from a list of chapters (heading, content) tuples.
+    * @param contents A list of (string, string) tuples, where the first string is the title and the other the content.
+    * @param outputFile The output file to which the markdown file is written.
+    */
+  def contentsToMarkdown(
+      contents: List[(String, String)],
+      outputFile: File
+  ): Unit = {
+    outputFile.getParentFile.mkdirs()
+    val fileWriter = new PrintWriter(outputFile)
+    for ((head, content) <- contents) {
+      fileWriter.println(head)
+      fileWriter.println()
+      fileWriter.println(content)
+      fileWriter.println()
+    }
+    fileWriter.close()
   }
+
+  /**
+    * Converts a resource to a file
+    * @param resource Which resource
+    * @param outputFile The output file
+    */
+  def resourceToFile(resource: String, outputFile: File): Unit = {
+    val source = getClass.getResourceAsStream(resource)
+    IoUtils.copyStreamToFile(source, outputFile, createDirs = true)
+  }
+
+  /**
+    * Generates the README
+    * @param outputFile The readme file
+    */
+  def generateReadme(outputFile: File): Unit = {
+    contentsToMarkdown(readmeContents, outputFile)
+  }
+
+  /**
+    * Outputs markdown documentation for LAIKA processing.
+    * @param outputDirectory outputs the Markdown documentation in this directory
+    */
+  def generateDocumentation(outputDirectory: File): Unit = {
+    outputDirectory.mkdirs()
+
+    contentsToMarkdown(mainPageContents,
+                       new File(outputDirectory + "/index.md"))
+    resourceToFile("/nl/biopet/utils/tool/default.template.html",
+                   new File(outputDirectory + "/default.template.html"))
+    resourceToFile("/nl/biopet/utils/tool/bootstrap.css",
+                   new File(outputDirectory + "/css/bootstrap.css"))
+    resourceToFile("/nl/biopet/utils/tool/docs.css",
+                   new File(outputDirectory + "/css/docs.css"))
```

Comment:
This can be writen better like this: `new File(outputDirectory, "css" + File.separator + "docs.css")`
An other option is:
```
val cssDir = new File(outputDirectory, "css")
val file = new File(cssDir, "docs.css")
```